### DISCUSSION TITLE
稳定后是否提交到Google Play商店？

### DISCUSSION THREAD

**Discussion by Losalado** [SagerNet/SagerNet#73] [Q&A] [answered]
Chromebook设备无法通过apk安装，未来是否会在应用市场上架？

**Comment by nekohasekai** [comment, marked as answer]
无法安装请提供详细日志, play 已经在公开测试轨道了.

**Comment by Losalado** [comment]
> 无法安装请提供详细日志, play 已经在公开测试轨道了.

不是apk的问题，是因为Chromebook只允许安装Google Play商店的应用，没有控制未知来源应用的开关

**Reply by nekohasekai** [reply]
https://play.google.com/store/apps/details?id=io.nekohasekai.sagernet

**Reply by Losalado** [reply]
太感谢了
